autoscale-core/src/main/java/com/hpe/caf/autoscale/core/AutoscaleScheduler.java

```diff
@@ -206,7 +206,7 @@ private void scheduleOrReschedule(final ScalingConfiguration config, final Workl
     private void cancel(final String id)
     {
         if ( scheduledServices.containsKey(id) ) {
-            LOG.debug("Cancelling service {}", id);
+            LOG.info("Cancelling service {}", id);
```

Comment:
What does cancelling service mean?

Reply:
Method documentation: 
     * Cancel the scheduling of a thread, and remove it from the map of monitored services.
     * @param id the name/reference of the service to stop monitoring

Reply:
Ok I am ok to log this at INFO then.